ui/.env.example

```diff
@@ -0,0 +1,5 @@
+IL_UI_ADMIN_USERNAME=admin
+IL_UI_ADMIN_PASSWORD=password
```

Comment:
i assume this is not the permanent password? :), otherwise we need some way to share this through some other way.

Reply:
It's at build time and temp until we get Oauth configured. The data is all open atm.